pkgs/servers/sql/postgresql/default.nix

```
@@ -33,6 +33,8 @@ let
       inherit sha256;
     };
 
+    hardeningEnable = [ "pie" ];
```

it fails on Darwin
```suggestion
    hardeningEnable = lib.optionals (!stdenv.isDarwin) [ "pie" ];
```

btw, this PR should target staging

---

Is the condition correct or do you need to  invert it?

---

Is `pie` not support on darwin by default?

---

AFAICT this isn't a darwin but an LLVM problem. Trying to build postgres on Linux with LLVM fails during configuring like this:
```
checking for __builtin_bswap16... yes
checking for __builtin_bswap32... yes
checking for __builtin_bswap64... yes
checking for __builtin_clz... yes
checking for __builtin_ctz... yes
checking for __builtin_popcount... yes
checking for _LARGEFILE_SOURCE value needed for large files... no
checking how cc reports undeclared, standard C functions... configure: error: in `/build/postgresql-13.3':
configure: error: cannot detect from compiler exit status or warnings
See `config.log' for more details
builder for '/nix/store/bg5yp1lc6kfyd0pnvmb369a2q9mh3wvx-postgresql-13.3.drv' failed with exit code 1
```

Overriding `hardeningEnable` to `[]` makes it build again.